### DISCUSSION TITLE
static import of css files fails

### DISCUSSION THREAD

**Discussion by anky2k** [vercel/next.js#26321] [Help]
### What version of Next.js are you using?

11

### What version of Node.js are you using?

12.20.1

### What browser are you using?

chrome

### What operating system are you using?

macOS

### How are you deploying your application?

next start

### Describe the Bug

facing this issues during next build

`./src/styles/global.css.webpack[javascript/auto]!=!./node_modules/next/dist/compiled/css-loader/cjs.js??ruleSet[1].rules[3].oneOf[6].use[1]!./node_modules/next/dist/compiled/postcss-loader/cjs.js??ruleSet[1].rules[3].oneOf[6].use[2]!./src/styles/global.css
TypeError: Cannot read property 'theme' of undefined`

have imported

src/styles/global.css

in pages/_app.js

### Expected Behavior

static imports shoudl work

### To Reproduce

import any .css file in _app.js

**Comment by anky2k** [comment]
next.config.js

`
const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
const withSourceMaps = require('@zeit/next-source-maps');
// const withPWA = require('next-pwa');

const {
  GEN_SOURCE_MAP,
  NODE_ENV,
  BASE_PATH,
  MOCK_MODE = 'n',
  APP_ENV = 'production'
} = process.env;

const dev = NODE_ENV !== 'production';
// const local = APP_ENV === 'local';
// const prod = NODE_ENV === 'production';
const genSourceMap = GEN_SOURCE_MAP === 'y';
const appVersion = require('./app-version');

// eslint-disable-next-line no-console
console.log(`running in ${dev ? 'dev' : 'production'} mode pointing to ${APP_ENV}`);

const nextConfig = {
  images: {
    domains: ['akamaividz2.zee5.com'],
  },
  // would suggest keeping this false
  trailingSlash: false, // https://github.com/zeit/next.js/issues/8119
  experimental: { // this takes the module/nomodule approach -  https://nextjs.org/blog/next-9-1#module--nomodule
    modern: true
  },
  i18n: {
    localeDetection: false,
    locales: ['en-in', 'hi-in', 'bn-in'],
    defaultLocale: 'en-in'
  },
  pwa: {
    swSrc: './src/service-worker.js',
    dest: 'public'
  },
  generateEtags: true,
  assetPrefix: BASE_PATH || '',
  publicRuntimeConfig: {
    basePath: BASE_PATH,
    mockMode: MOCK_MODE,
    appEnv: APP_ENV
  },
  generateBuildId: async () => appVersion,
  webpack: config => {
    config.output.publicPath = '';
    config.resolve.fallback = { fs: false };
    config.plugins.push(new BundleAnalyzerPlugin({
      openAnalyzer: false,
      generateStatsFile: true,
      analyzerMode: 'json',
      excludeAssets: ['node_modules', 'coverage', 'tools', 'public', 'perf'],
      statsOptions: {
        exclude: ['node_modules', 'coverage', 'tools', 'public', 'perf'],
        hash: false,
        cached: false,
        cachedAssets: false,
        reasons: false,
        source: true,
        warnings: false,
        errors: false,
        errorDetails: false,
        publicPath: false,
        modulesSort: 'size',
        chunksSort: 'size',
        assetsSort: 'size'
      }
    }));    
    return config;
  }
};

// eslint-disable-next-line no-nested-ternary
// module.exports = genSourceMap ? withSourceMaps(nextConfig) : (!local ? withPWA(nextConfig) : nextConfig);
module.exports = genSourceMap ? withSourceMaps(nextConfig) : nextConfig;
`

package.json

`
{
  "name": "@zee5/webapp",
  "version": "1.0.0",
  "main": "pages/_app.js",
  "sideEffects": false,
  "browserslist": [
    ">0.3%",
    "not ie 11",
    "not dead",
    "not op_mini all"
  ],
  "dependencies": {
    "@babel/runtime-corejs2": "^7.8.7",
    "@date-io/date-fns": "^1.3.6",
    "@lhci/cli": "^0.7.2",
    "@tailwindcss/line-clamp": "^0.2.0",
    "@testing-library/react": "^10.0.4",
    "@zeit/next-source-maps": "0.0.3",
    "babel-plugin-module-resolver": "^4.0.0",
    "can-use-dom": "^0.1.0",
    "date-fns": "^2.19.0",
    "eslint-config-airbnb": "^18.1.0",
    "eslint-plugin-babel": "^5.3.0",
    "eslint-plugin-jest": "^23.8.2",
    "jest-fetch-mock": "^3.0.3",
    "local-web-server": "^4.2.0",
    "lodash": "^4.17.21",
    "miragejs": "^0.1.41",
    "mq-polyfill": "^1.1.8",
    "next": "^11.0.0",
    "next-pwa": "^5.0.5",
    "next-seo": "^4.23.0",
    "parse-link-header": "^1.0.1",
    "prop-types": "^15.7.2",
    "raw-loader": "^3.1.0",
    "react": "^17.0.2",
    "react-dom": "^17.0.2",
    "react-error-boundary": "^3.0.2",
    "react-scripts": "^4.0.0",
    "serve": "^11.3.0",
    "styled-jsx-css-loader": "^0.3.0",
    "tailwind-scrollbar-hide": "^1.0.3",
    "webpack-bundle-analyzer": "^4.4.2",
    "workbox-expiration": "^6.1.1",
    "workbox-precaching": "^6.1.1",
    "workbox-routing": "^6.1.1",
    "workbox-strategies": "^6.1.1",
    "workbox-window": "^6.1.0"
  },
  "devDependencies": {
    "@babel/cli": "^7.5.0",
    "@babel/core": "^7.14.3",
    "@babel/eslint-parser": "^7.14.3",
    "@babel/plugin-proposal-class-properties": "^7.4.4",
    "@babel/preset-env": "^7.5.2",
    "@babel/preset-modules": "^0.1.1",
    "@svgr/webpack": "^5.5.0",
    "@tailwindcss/jit": "^0.1.17",
    "@testing-library/jest-dom": "^4.1.0",
    "@testing-library/react-hooks": "^3.7.0",
    "@zeit/next-css": "^1.0.1",
    "autoprefixer": "^10.2.6",
    "babel-core": "7.0.0-bridge.0",
    "babel-eslint": "^10.0.1",
    "babel-jest": "^24.0.0",
    "babel-loader": "^8.1.0",
    "babel-plugin-groundskeeper-willie": "^1.3.2",
    "babel-plugin-inline-dotenv": "^1.1.2",
    "babel-plugin-lodash": "^3.3.4",
    "chalk": "^4.1.0",
    "child_process": "^1.0.2",
    "commander": "^7.1.0",
    "cross-env": "^5.2.1",
    "eslint": "^7.27.0",
    "eslint-config-next": "^11.0.0",
    "eslint-plugin-import": "^2.20.2",
    "eslint-plugin-jsx-a11y": "^6.2.3",
    "eslint-plugin-node": "^11.1.0",
    "eslint-plugin-promise": "^4.2.1",
    "eslint-plugin-react": "^7.20.0",
    "eslint-plugin-react-hooks": "^4.0.1",
    "eslint-plugin-tree-shaking": "^1.7.4",
    "file-loader": "^6.0.0",
    "fs": "0.0.1-security",
    "jest": "^24.0.0",
    "jest-dom": "^4.0.0",
    "jest-html-reporter": "^2.5.0",
    "jest-junit": "^10.0.0",
    "jest-svg-transformer": "^1.0.0",
    "lint-staged": "^8.1.0",
    "next-compose-plugins": "^2.2.0",
    "nodemon": "^1.12.1",
    "parse-git": "^1.1.2",
    "path": "^0.12.7",
    "postcss": "^8.3.5",
    "pre-commit": "^1.2.2",
    "raf": "^3.4.0",
    "react-perf-devtool": "^3.1.8",
    "react-test-renderer": "^16.13.1",
    "rimraf": "^2.6.2",
    "source-map-explorer": "^2.4.2",
    "tailwindcss": "^2.2.0",
    "url-loader": "^4.1.1",
    "webpack-clear-console": "^1.0.3"
  },
  "scripts": {
    "copy-files": "node ./tools/copy-files-to-build.js",
    "lint-staged": "lint-staged",
    "analyze": "npm run clean && cross-env GEN_SOURCE_MAP=y  NODE_ENV=production next build && node ./tools/bundle-analyzer.js",
    "test:ci": "jest --maxWorkers=2 --ci",
    "test:no-watch": "jest",
    "test": "npm run test:no-watch -- --watch --no-cache",
    "test:coverage": "npm run test:no-watch -- --coverage --colors",
    "lint": "next lint",
    "lint:fix": "npm run lint -- --fix",
    "local": "cross-env APP_ENV=local next",
    "dev": "cross-env APP_ENV=development next",
    "debug": "cross-env APP_ENV=local NODE_OPTIONS='--inspect' next dev",
    "mdebug": "cross-env APP_ENV=local MOCK_MODE=y NODE_OPTIONS='--inspect' next dev",
    "staging": "cross-env APP_ENV=staging next",
    "production": "cross-env APP_ENV=production next",
    "mock": "cross-env APP_ENV=production MOCK_MODE=y npm run dev",
    "build": "cross-env NODE_ENV=production next build",
    "preexport": "npm run clean && npm run build",
    "export": "next export",
    "export-mock": "cross-env MOCK_MODE=y npm run export",
    "prestart": "npm run clean && npm run build",
    "start": "cross-env NODE_ENV=production next start",
    "prestart-mock": "cross-env APP_ENV=development MOCK_MODE=y npm run clean && npm run build",
    "start-mock": "cross-env APP_ENV=development MOCK_MODE=y next start",
    "clean": "rimraf node_modules/.cache .next",
    "preserve": "cross-env MOCK_MODE=y  npm run export",
    "gitlint": "git log -1 --pretty=%B | gitlint",
    "multilingual-build": "node ./command.js",
    "serve": "ws --directory out --static.maxage 1000000 --static.extensions html --static.index index.html --http2 --compress"
  },
  "pre-commit": [
    "lint-staged",
    "test:no-watch",
    "analyze"
  ],
  "post-commit": [
    "gitlint",
    "git add .",
    "git commit -am 'append bundle report' --no-verify"
  ],
  "lint-staged": {
    "/**/*.{js,jsx,tsx,ts}": [
      "npm run lint:fix",
      "git add"
    ]
  },
  "engines": {
    "node": ">= 14"
  },
  "author": "Ankit Agarwal (ankit.agarwal@ril.com)",
  "license": "ISC"
}

`

**Comment by Miikis** [comment]
Had this same unhelpful error when migrating from tailwind v1 to tailwind v2... at the same time as migrating nextjs v9 to nextjs v11, where we also switched from node-sass to dart-sass. It turned out to be just a missing semi-colon 💀 but make sure you're doing everything in the [tailwind v2 migration guide](https://tailwindcss.com/docs/upgrading-to-v2#update-typography-and-forms-plugins)... and, if possible, make sure your postcss and tailwind config look like the [examples](https://github.com/vercel/next.js/tree/canary/examples/with-tailwindcss)

**Reply by Miikis** [reply]
It was technically not _the same_ error I had, but it was very similar
